src/home-staging/replace-furniture.js

```diff
@@ -0,0 +1,139 @@
+import getFurnitureInfo  from '../furniture/get-info.js'
+import callService  from '../utils/services/call.js'
+import normalizeFurnitureInfo from '../furniture/common/normalize-furniture-info.js'
+
+var userQuery, searchCount, margin, furnitureInfo, position, rotation
+
+var config = {
+  'default_margin': 0.1,
+  'default_search': '-generic isPublished:true',
+  'tag_black_list': [
+    'simplygon',
+    'hasChangeableMaterials',
+    'autofurnish',
+    'wallAttached',
+    'daybed',
+    'oval',
+    'rectangular',
+    '2 seater',
+    '3 seater',
+    '4 seater'
+  ],
+}
+
+export default function replaceFurniture (id, options) {
+  // API
+  options = options || {}
+  userQuery = options.query || null
+  position = options.position || {x: 0, y: 0, z: 0}
+  rotation = options.rotation || {x: 0, y: 0, z: 0}
+  // config publishable api key
```

Comment:
Is that supposed to *not* happen? I think this method might as well be publishable-key'd, no?